### PR TITLE
Disabled `send_page_view` to get rid of events duplication in ga4 plugin

### PR DESCRIPTION
**Pull request by ekrasnopolskaia**
## Hey, I just made a Pull Request!

Due to the fact that the default invents with send_page_view are sent, it turns out that the sending happens by default and from the plugin itself. And due to the fact that the default events cannot collect uid, there is a duplication (event with and without uid).

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by sramakr** [comment]
Hello @freben yes, I will take a look. Thank you.

**Comment by sramakr** [comment]
Looks good to me 👍

File: plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export class GoogleAnalytics4 implements AnalyticsApi {
       },
       gtagOptions: {
         debug_mode: debug,
+        send_page_view: false,
```
Comment (sramakr) [inline plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts:73]: if we can make this configurable, that would great!
Like how we did with debug, default value can be false but, allows the users to customize if we need it this value to be true at some point.

**Review by sramakr — CHANGES_REQUESTED** [review]

File: plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts (hunk 1 above)
Comment (ekrasnopolskaia) [reply plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts]: Done! also tried to describe it to the readme

File: plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts [hunk 2]
```diff
@@ -112,6 +115,9 @@ export class GoogleAnalytics4 implements AnalyticsApi {
     const identity =
       config.getOptionalString('app.analytics.ga4.identity') || 'disabled';
     const debug = config.getOptionalBoolean('app.analytics.ga4.debug') ?? false;
+    const enableSendPageView =
+      config.getOptionalBoolean('app.analytics.ga4.enableSendPageView') ??
```
[the remark this answers is not in the document]
Comment (ekrasnopolskaia) [reply plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts:119]: Added! but now I see aт E2E Linux cheks error, can you help me please? could the config affect this?

[the remark this answers is not in the document]
Comment (sramakr) [reply plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts:119]: I don't believe this error is related to config schema change. I see this error in other PR's as well. Error is coming from emotion package.

**Review by freben — APPROVED** [review]

**Pull request merged**